README.md

```diff
@@ -8,17 +8,19 @@ on the browser.
 
 ## How to use this repo
 
-* Clone it, and `cd` into it
-* `npm install`
-* `npm start`
+1. Clone it, and `cd` into it
+2. Install [Node.js](https://nodejs.org/en/).
+3. Install Node Package Manager `sudo npm install npm -g`.
```

Comment:
I think we should remove points 2 and 3 because anyone who works on this repo should already be familiar with Node and npm.